src/test/lib/pgSQL-tests.js

```diff
@@ -0,0 +1,425 @@
+/**************************************************************
+ *
+ * UNIT TESTS postgresQL
+ *
+ **************************************************************/
+
+// external modules
+const assert = require('assert');
+const config = require('../../config/config');
+const async = require('async');
+pg=require('../../lib/postgresQL')(config.pg);
+var createTable=[
+    `CREATE TABLE IF NOT EXISTS repositories (id serial PRIMARY KEY, name varchar NOT NULL,
+        domain varchar NOT NULL, contact varchar NOT NULL, token varchar NOT NULL);`,
+    'CREATE INDEX IF NOT EXISTS repositories_name_idx ON repositories(name);',
+    'CREATE INDEX IF NOT EXISTS repositories_domain_idx ON repositories(domain);',
+    'CREATE INDEX IF NOT EXISTS repositories_contact_idx ON repositories(contact);',
+    'CREATE INDEX IF NOT EXISTS repositories_token_idx ON repositories(token);'];
+var dropTable=[`DROP TABLE repositories;`];
+var querystr='';
+
+describe('postgresQL.js: db methods unit tests.', function () {
+
+    describe('Require module.', function () {
+
+        it ('Should not throw an error when requiring the module', function (done) {
+            assert.doesNotThrow(function () {
+                pg;
+            });
+            done();
+        });
+
+        it ('Should be an object--SQL statement', function (done) {
+            assert.ok(typeof pg === 'object');
+            done();
+        });
+
+        it ('Should be able to execute a given statement with no seperate params, Create/intialize the tables', function (done) {
+            async.eachSeries(
+                createTable,
+                (createTable, xcallback) => {
+                    pg.execute(
+                        createTable, [],
+                        (err,res) => {
+                             xcallback(err, res);
+                        }
+                    );
+                },
+                () => { 
+                 done();
+                 }
+            );
+        });
+
+
+    });
+
+    before(function (done) { // create/intialize db
+        
+        done();
+    });
+
+    after(function (done) { // clean/erase db
+        async.eachSeries(
+            dropTable,
+            (dropTable, xcallback) => {
+                pg.execute(
+                    dropTable, [],
+                    (err,res) => {
+                         xcallback(err, res);
+                    }
+                );
+            },
+            () => { pg.close(); done();}
```

Comment:
spacing after `done()`
```suggestion
            () => { pg.close(); done(); }
```